packages/ses/index.d.ts

```diff
@@ -115,12 +115,12 @@ interface ToStringable {
 }
 
 export interface Assert {
-  (value: any, details?: Details, errorConstructor?:ErrorConstructor): void;
+  (value: any, details?: Details, errorConstructor?:ErrorConstructor): asserts value;
   typeof: AssertTypeof,
   error(details?: Details, errorConstructor?:ErrorConstructor): Error;
   fail(details?: Details, errorConstructor?:ErrorConstructor): never;
   equal(left: any, right: any, details?: Details, errorConstructor?:ErrorConstructor): void;
```

Comment:
Btw, in the idea of tightening the assertions, node types equal like this:

```
strictEqual<T>(actual: any, expected: T, message?: string | Error): asserts actual is T;
```

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/3c27f1284c29971e9efd2f69dc0b86e47ee432aa/types/node/assert.d.ts#L66

Another approach might be a type equality check: https://stackoverflow.com/a/53808212

```
equal<T, U>(left: T, right: U):  (<G>() => G extends T ? 1 : 2) extends (<G>() => G extends U ? 1 : 2) ? void : never;
```

Reply:
The `strictEqual` assertion is elegant.